docs-search/client-halogen/src/Docs/Search/BrowserEngine.js

```diff
@@ -0,0 +1,29 @@
+/* global exports */
```

Comment:
Git doesn't detect this file as a rename, for some reason. But it is.

Reply:
Did you use `git mv`?